### PR TITLE
No flake8 E123 left in Tests/

### PR DESCRIPTION
**Pull request by peterjc**
This pull request continues on going flake8/black style work

- [X] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [X] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [X] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

**Comment by peterjc** [comment]
Evidently I was not run running flake8 exactly the same way locally...

```
$ flake8
./Tests/test_CAPS.py:56:13: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_NaiveBayes.py:45:13: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_NaiveBayes.py:58:13: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_KGML_graphics_online.py:64:13: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_SeqIO_write.py:64:5: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_KGML_graphics.py:84:13: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_SeqIO_Xdna.py:36:21: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_SeqIO_Xdna.py:43:21: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_SeqIO_Xdna.py:44:17: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_SeqIO_Xdna.py:45:13: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_SeqIO_Xdna.py:61:21: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_SeqIO_Xdna.py:68:21: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_SeqIO_Xdna.py:69:17: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_SeqIO_Xdna.py:70:13: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_SeqIO_Xdna.py:86:21: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_SeqIO_Xdna.py:93:21: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_SeqIO_Xdna.py:94:17: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_SeqIO_Xdna.py:95:13: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_SeqIO_Xdna.py:96:9: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_MarkovModel.py:47:13: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_MarkovModel.py:125:13: E123 closing bracket does not match indentation of opening bracket's line
```

**Comment by peterjc** [comment]
Simple brute force solution, run black on the problem files - cross reference #2552

**Comment by peterjc** [comment]
This now overlaps with #2597 but that will need updating anyway due to a merge conflict :(

**Comment by peterjc** [comment]
@svalqui / @MarkusPiotrowski would you like to look at / review this please?

**Comment by peterjc** [comment]
@svalqui I think there has been a misunderstanding - I wanted you to review my changes on #2871 (running black etc), not make a bunch of new pull requests fixing E123 by hand.

Well, now that you have done them, I will review them and apply them, and then update my #2871 PR.

**Comment by peterjc** [comment]
This is looking much better now that @svalqui fixed a bunch of E123 by hand:

```
$ flake8
./Tests/test_CAPS.py:56:13: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_NaiveBayes.py:45:13: E123 closing bracket does not match indentation of opening bracket's line
./Tests/test_NaiveBayes.py:58:13: E123 closing bracket does not match indentation of opening bracket's line
```

**Comment by peterjc** [comment]
Thanks @svalqui - it wasn't what I expected, but by fixing the E123 cases only this resulted in a much smaller set of changes compared to my original pull request (using black which made other style changes as well).

**Comment by svalqui** [comment]
With these 2 the E123 should go away , sorry for the misunderstanding, I'm finding hard to follow the work flow on this current times.
This PR is fine you should merge it, the last 2 PR of mine here you can merge or ignore/close and take yours, either way is fine.
Now I'll try to fix conflicts on PRs 2597, 2739 and 2753, in the following 4 to 6 days, I wanted to fix them before the next release, but if not I'll do it later.

**Comment by peterjc** [comment]
Thank you :)
